build.gradle.kts

```diff
@@ -163,7 +163,11 @@ allprojects {
       // Default use MiniGravitino to run integration tests
       param.environment("GRAVITINO_ROOT_DIR", project.rootDir.path)
       param.environment("IT_PROJECT_DIR", project.buildDir.path)
-      param.environment("HADOOP_USER_NAME", "anonymous")
+      if (project.name == "authorization-ranger") {
+        param.environment("HADOOP_USER_NAME", "test")
+      } else {
+        param.environment("HADOOP_USER_NAME", "anonymous")
+      }
```

Comment:
I think we can user-define environment variable `HADOOP_USER_NAME`  in the submodule

```
      // If the environment variable `HADOOP_USER_NAME` is not customized in submodule,
      // then set it to "anonymous"
      if (param.environment["HADOOP_USER_NAME"] == null) {
        param.environment("HADOOP_USER_NAME", "anonymous")
      }
```

and add these code in the `authorization-ranger` submodule 
```
tasks.test {
  ......

  doFirst {
    environment("HADOOP_USER_NAME", "test")
  }
}
```

Reply:
Done.